src/pages/registration/references_tab/sub_type_forms/BookForm.tsx

```diff
@@ -90,26 +102,19 @@ const BookForm: FC = () => {
       <div>
         <StyledTypography variant="h5">{t('references.series')}</StyledTypography>
         <Typography>{t('references.series_info')}</Typography>
+
         <Field name={ReferenceFieldNames.SERIES_TITLE}>
           {({ field: { name, value } }: FieldProps) => (
-            <>
-              <PublicationChannelSearch
-                dataTestId="autosearch-series"
-                clearSearchField={value === ''}
-                label={t('common:title')}
-                publicationTable={PublicationTableNumber.PUBLICATION_CHANNELS}
-                setValueFunction={(inputValue) => setFieldValue(name, inputValue.title ?? '')}
-                placeholder={t('references.search_for_series')}
-              />
-              {value && (
-                <PublisherRow
-                  dataTestId="autosearch-results-series"
-                  label={t('common:title')}
-                  publisher={{ title: value }}
-                  onClickDelete={() => setFieldValue(name, '')}
-                />
-              )}
-            </>
+            <PublisherField
```

Comment:
dette blir ikke PublisherField, men SeriesField...

her må vi sette seriesTitle, og ikke title - for hvis vi skriver title, så overskriver vi publisher 🙃

Reply:
https://github.com/BIBSYSDEV/nva-datamodel-java/blob/develop/documentation/BookAnthology.json#L69